### PR TITLE
feat(preset): add prettier and standard to linters package

### LINKED ISSUE #17267: Add Prettier and Standard to the `packages:linter` preset
### What would you like Renovate to be able to do?

How about adding Prettier and Standard to the `packages:linters` preset, like this:

```diff
{
  "extends": [
    "packages:emberTemplateLint",
    "packages:eslint",
    "packages:stylelint",
    "packages:tslint",
+   "packages:prettier",
+   "packages:standard"
  ],
  "matchPackageNames": [
    "remark-lint"
  ]
}
```

The description for the `packages:linters` says:

> All lint-related packages.

But it doesn't include all lint related packages, because it's missing Prettier, and Standard.

### If you have any ideas on how this should be implemented, please tell us here.

Add some more lint-related packages to the `packages:linters` preset. Not sure if we want to create a `packages:prettier` and `packages:standard` which would only have one package name each?

### Is this a feature you are interested in implementing yourself?

No

### PR DESCRIPTION
**Pull request by adarshlilha**
## Changes
Adds Prettier and Standard to the packages:linter preset
Closes #17267 

## Context
Closes #17267 

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/presets/internal/packages.ts
Comment (rarkins) [inline lib/config/presets/internal/packages.ts:41]: Is there more than one package? Because this will match `standard`, `standardxyz`, `standard-abc`, etc.

File: lib/config/presets/internal/packages.ts
Comment (rarkins) [inline lib/config/presets/internal/packages.ts:50]: Same question as above.

Comment (HonkingGoose) [reply lib/config/presets/internal/packages.ts:50]: I think we should use [`matchPackageNames`](https://docs.renovatebot.com/configuration-options/#matchpackagenames) instead, like this:

```suggestion
    matchPackageNames: ['prettier'],
```

This avoids matching `prettier-plugin` or similar names.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

**Review by JamieMagee — CHANGES_REQUESTED** [review]
Re-add the `eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion` comments please

**Comment by adarshlilha** [comment]
> Re-add the `eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion` comments please

sorry, my bad, have added back the comments

**Review by HonkingGoose — APPROVED** [review]
I think this looks okay, but I'll let a maintainer review it also. 😉

File: lib/config/presets/internal/packages.ts
Comment (viceice) [inline lib/config/presets/internal/packages.ts:62]: can't we simply add those two packages to this matcher?

Comment (rarkins) [reply lib/config/presets/internal/packages.ts:62]: To line 62?

Comment (rarkins) [reply lib/config/presets/internal/packages.ts:62]: Yes, good idea

**Pull request closed without merging**
